Umbraco-Cloud/Set-Up/Working-With-Visual-Studio/index.md

```diff
@@ -30,7 +30,7 @@ Before running the UaaS.cmd tool you will need the git clone url for your Umbrac
 
 ![](images/connect-my-machine.png)
 
-Running the UaaS.cmd tool will download the latest Visual Studio generator (waasp.exe) and then prompt you to enter the git clone url for your Project.
+Running the UaaS.cmd tool will download the latest Visual Studio generator (wasp.exe) and then prompt you to enter the git clone url for your Project.
```

Comment:
This is an incorrect change.  If you inspect the UAAS.cmd tool, you will notice that it downloads `waasp.exe`.